watertap/unit_models/ion_exchange_0D.py

```diff
@@ -1051,10 +1026,16 @@ def eq_bed_flow(b):
 
         @self.Constraint(doc="Column height")
         def eq_col_height(b):
-            return (
-                b.col_height
-                == b.bed_depth + b.distributor_h + b.underdrain_h + b.bed_expansion_h
-            )
+            if self.config.regenerant != RegenerantChem.single_use:
+                return (
+                    b.col_height
+                    == b.bed_depth
+                    + b.distributor_h
+                    + b.underdrain_h
+                    + b.bed_expansion_h
+                )
+            else:
+                return b.col_height == b.bed_depth + b.distributor_h + b.underdrain_h
```

Comment:
Does this make more sense than fixing bed_expansion_h to 0?

Reply:
Well ``bed_expansion_h`` is an ``Expression`` that is a function of other Params so would require setting those also to zero. Would you prefer that rather than the conditionals?

Reply:
I'm just thinking of it from a user perspective. If the model was preconstructed and then toggled between `single_use` and not, is it more or less intuitive if:

1. Previous variables that did exist on the model are no longer there, calling them will result in an error
2. Previous variables that were nonzero are now set to zero (or 1 depending on the equation structure), users have to interpret this as inactive when `single_use`

I don't know the best answer, but this effects a few variables throughout the condition